x/auction/types/msg.go

```diff
@@ -35,14 +28,15 @@ func (msg MsgPlaceBid) Type() string { return "place_bid" }
 
 // ValidateBasic does a simple validation check that doesn't require access to state.
 func (msg MsgPlaceBid) ValidateBasic() error {
-	if msg.AuctionID == 0 {
+	if msg.AuctionId == 0 {
 		return errors.New("auction id cannot be zero")
 	}
-	if msg.Bidder.Empty() {
+	if len(msg.Bidder) == 0 {
```

Comment:
fyi, `AccAddressFromBech32` handles address length validation so don't need to include them

Reply:
https://github.com/Kava-Labs/kava/pull/1052/commits/e3c0f253bd10ae79005926603c17e065303e1eb3